exercises/pig-latin/canonical-data.json

```diff
@@ -137,6 +131,29 @@
       ]
     },
     {
+      "description": "position of y in a word determines if it is a consonant or a vowel",
+      "cases": [
+        {
+          "description": "y at beginning of a word is treated like a consonant",
```

Comment:
```
"y is treated like a consonant at the beginning of a word "
```

Reply:
@rpottsoh Could you please elaborate, why this is necessary? (same for the other comment)
I am a bit confused about why this matters and what syntax we should aim for. Is there a guideline somewhere?

Reply:
@Vankog the sentence was lacking a comma.  By reordering the wording it simplifies the statement and removes the need for a comma.  Also, several tracks employ test generators.  Often times `"description"` becomes the method name of the test.

Reply:
I see, thanks